package/yast2-packager.changes

```diff
@@ -1,3 +1,8 @@
+Thu Nov  9 14:45:15 UTC 2017 - knut.anderssen@suse.com
```

Comment:
IIRC the change entry should start with the `-----` separator on the first line.

Reply:
yep, I realized it yesterday when cheking, an also the bug reference is wrong needs to fix it, but was working in yast2-network (remote) changes related to this PR.

Thnx for checking!!